src/components/data-pane/field-list.tsx

```diff
@@ -76,7 +133,8 @@ export const FieldList = connect(
 export const PresetWildcardFieldList = connect(
   (state: State) => {
     return {
-      fieldDefs: getPresetWildcardFields(state)
+      fieldDefs: getPresetWildcardFields(state),
+      schema: state.present.dataset.schema
```

Comment:
Should we use the selector for `getSchema` here?  I guess we should for consistency